dev/config/ory/kratos.yml

```diff
@@ -132,19 +132,6 @@ selfservice:
             #         name: Authorization
             #         value: The-Value-of-My-Key
             #         in: header
```

Comment:
you could probably remove the 15 lines above also

Reply:
https://github.com/GaloyMoney/galoy/pull/3925/commits/ff8786f622994e59d6171cdcf2eac4bdf352432f